README.md

```diff
@@ -1,45 +1,102 @@
-## hivemind: decentralized deep learning in PyTorch
+## Hivemind: decentralized deep learning in PyTorch
+
 [![Build status](https://circleci.com/gh/learning-at-home/hivemind.svg?style=shield)](https://circleci.com/gh/learning-at-home/hivemind)
 [![Documentation Status](https://readthedocs.org/projects/learning-at-home/badge/?version=latest)](https://learning-at-home.readthedocs.io/en/latest/?badge=latest)
 [![Gitter](https://badges.gitter.im/learning-at-home/hivemind.svg)](https://gitter.im/learning-at-home/hivemind?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge)
 
-Hivemind is a PyTorch library to train large neural networks across the Internet. Imagine training one huge Transformer model
-  on thousands of computers from different universities, companies, and volunteers.
+Hivemind is a PyTorch library to train large neural networks across the Internet. Its intended usage is training a
+single Transformer model on hundreds of computers from different universities, companies, and volunteers.
```

Comment:
technically speaking, we did run a 1250~1300 node scalability experiment :) Though, it's okay if you prefer it that way

Reply:
Right, but no Transformer training experiment on thousands of diverse computers yet :) Let's keep the claims modest for the time being